tools/tasks.py

```diff
@@ -674,7 +674,7 @@ def make_os_tasks(processes):
         UnixTask("couchbase user definition", "getent passwd couchbase"),
         UnixTask("couchbase user limits", "su couchbase -c \"ulimit -a\"",
                  privileged=True),
-        UnixTask("couchbase user limits", "su couchbase -c \"ulimit -a\"",
+        UnixTask("sync_gateway user limits", "su sync_gateway -c \"ulimit -a\"",
```

Comment:
Should there be a 'sync gateway user definition' task as well, similar to the couchbase one above?

Reply:
Yeah, good point, added.